app/src/main/java/com/example/playlistmaker/data/dto/ImageLoaderGlide.kt

```diff
@@ -0,0 +1,20 @@
+package com.example.playlistmaker.data.dto
+
+import android.widget.ImageView
+import com.bumptech.glide.Glide
+import com.bumptech.glide.load.resource.bitmap.RoundedCorners
+import com.example.playlistmaker.domain.api.ImageLoader
+
+class ImageLoaderGlide: ImageLoader {
+
+    override fun loadImage(url: String, placeholder: Int, imageView: ImageView, round: Int) {
```

Comment:
```suggestion
    override fun loadImage(url: String, placeholder: Int, imageView: ImageView, @Px cornerRadius: Int) {
```
Так будет понятнее за что именно отвечает 4й параметр и в каком формате используется